### PR TITLE
TransactionGenerator proper implementation (#EC-149)

### PR DESCRIPTION
**Pull request by oskin1**

### CODE REVIEW

File: src/main/scala/encry/view/state/UtxoState.scala [hunk 1]
```diff
@@ -103,7 +103,8 @@ class UtxoState(override val version: VersionTag,
     log.debug(s"Generating proof for ${txs.length} transactions ...")
     val rootHash: ADDigest = persistentProver.digest
     if (txs.isEmpty) Failure(new Exception("Got empty transaction sequence"))
```
Comment (kapinuss) [inline src/main/scala/encry/view/state/UtxoState.scala]: Do you mean here to make a new instance of class Exception or to throw a new Exception? If second this code is wrong. def proofsForTransactions can return Failure to val (adProof: SerializedAdProof, adDigest: ADDigest) = state.proofsForTransactions(txs).get.

File: src/main/scala/encry/view/state/UtxoState.scala [hunk 2]
```diff
@@ -102,13 +102,10 @@ class UtxoState(override val version: VersionTag,
   def proofsForTransactions(txs: Seq[EncryBaseTransaction]): Try[(SerializedAdProof, ADDigest)] = {
     log.debug(s"Generating proof for ${txs.length} transactions ...")
     val rootHash: ADDigest = persistentProver.digest
-    if (txs.isEmpty) {
-      Failure(new Exception("Got empty transaction sequence"))
-    } else if (!storage.version.exists(_.sameElements(rootHash))) {
+    if (txs.isEmpty) Failure(new Exception("Got empty transaction sequence"))
+    else if (!storage.version.exists(_.sameElements(rootHash)))
       Failure(new Exception(s"Invalid storage version: ${storage.version.map(Algos.encode)} != ${Algos.encode(rootHash)}"))
```
Comment (kapinuss) [inline src/main/scala/encry/view/state/UtxoState.scala]: Do you mean here to make a new instance of class Exception or to throw a new Exception? If second this code is wrong. def proofsForTransactions can return Failure to val (adProof: SerializedAdProof, adDigest: ADDigest) = state.proofsForTransactions(txs).get.

File: src/main/resources/application.conf [hunk 3]
```diff
@@ -37,8 +37,10 @@ encry {
     useCli = true
   }
   testing {
-    transactionGeneration = false
-    keepPoolSize = 100
+    transactionGeneration = true
```
Comment (kapinuss) [inline src/main/resources/application.conf]: transactionGeneration = true switches on creating random txs on every node. Is it your goal?

Comment (oskin1) [reply src/main/resources/application.conf]: It allows you to test node properly every time you start the node (At the development stage). No doubt, we will set it to `false` in prod.

File: src/main/scala/encry/view/state/UtxoState.scala (hunk 2 above)
Comment (oskin1) [reply src/main/scala/encry/view/state/UtxoState.scala]: It was implied to work with exceptions within `Try` here if the failure occurred. `.get` from miner is already removed in another pr.

**Pull request merged**